BlackNectar/BlackNectar/Integers+.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Integers+.swift
+//  BlackNectar
+//
+//  Created by Cordero Hernandez on 12/29/16.
+//  Copyright © 2016 Black Whole. All rights reserved.
```

Comment:
The copyrights here are still showing Black Whole. We need to update the project to reflect `BlackSource`.